src/routers/OrganizationRouter.ts

```diff
@@ -349,12 +368,31 @@ const getAdminRouter = (basePath: string = '/', routePath: string = '/management
         throw new ParameterRequiredError('Invalid format for field: slug', 400);
       }
 
+      const tempData: { email: string; error?: string; status?: number }[] = [];
+
       const ownerIds = await forEachAsync(owners, async (email) => {
-        const user = await authMgmtService.getUserByEmail(email);
-        logger.debug(`resolved owner ${user.user_id} for email: ${email}`);
-        return user.user_id;
+        try {
+          const user = await authMgmtService.getUserByEmail(email);
+          logger.debug(`resolved owner ${user.user_id} for email: ${email}`);
+          tempData.push({ email: user.email, status: 200 });
+          return user.user_id;
+        } catch (err) {
+          logger.debug(`unresolved owner for email: ${email}`);
+          logger.error(err.message);
+          tempData.push({ email: email, error: err.message, status: err.code });
+        }
       });
 
+      if (tempData.some((res) => ![200, 409].includes(res.status))) {
+        const code = 403;
+        const response = createSerializer().serialize({ errors: tempData });
```

Comment:
We should be using the same serializer we are using for `PUT /management/users` -> createUserBulkSerializer:
```
import { createBulkSerializer as createUserBulkSerializer } from '../serializers/UserSerializer';

const code = 400;
const response = createUserBulkSerializer().serialize(data);
```